### DISCUSSION TITLE
Working on Paris 54148

### DISCUSSION THREAD

**Discussion by petemachin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#656] [Show and tell]
Hello, 
I just thought I'd share that v2b board with the red TTL level shifter and using the 'ambitious setup' with resistors and caps works straight out of the box with this egg shaped model. The next job is to mod the board with a DS18B20 ambient temperature sensor and see if I can get that working too. 

Thanks Thomas! (VisualApproach)

![IMG_0754](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/68728476/0591ea66-8d55-43dc-8801-03c89c9d06e9)
![IMG_0755](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/68728476/b3b87c41-6c72-4b0d-8341-d94d0624b7a5)

**Comment by visualapproach** [comment]
Nice! And thank you for your contribution ☕️ 
Have a great day!
